### PR TITLE
Update contributing.md file. Refs #756

### PR DESCRIPTION
**Pull request by koenedaele**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,11 +1,21 @@
 Contributing
 ============
 
-Atramhasis is being developed as open source software by the 
+Atramhasis is an open source SKOS editor developed by
```
Comment (JonasGe) [inline CONTRIBUTING.md:4]: The `the` is missing between `by` and `[Flanders...`

**Review by JonasGe — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (koenedaele) [reply CONTRIBUTING.md:4]: It felt weird to have that the there, so I deleted it on purpose. But I'm not a native English speaker, so I might be wrong.

Comment (JonasGe) [reply CONTRIBUTING.md:4]: Neither am I, I asked ChatGPT.

Comment (koenedaele) [reply CONTRIBUTING.md]: I worked around the issue by dropping the agency bit. Think even ChatGPT should be satisfied now :-)

**Review by JonasGe — APPROVED** [review]

**Pull request merged**